app/views/transfer/_empty.html.erb

```diff
@@ -0,0 +1,3 @@
+<tr class="empty">
+  <td colspan="6">There were no transfers matching your parameters.
```

Comment:
`td` not closed

Reply:
*smacks head* D'oh! I'm going to push something to fix this.